### PR TITLE
Host dashboard

### PR DESCRIPTION
**Pull request by RamyAlshurafa**
Relates #126

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,9 +9,11 @@ cache:
     - node_modules
 
 env:
-  - MONGODB=4.0.0
+  global:
+    - MONGODB=4.0.0
+    - MONGOURIATLAS=mongodb+srv://abd:123@localhost-yaadf.mongodb.net/atlas-test?retryWrites=true&w=majority
```
Comment (ashatat) [inline .travis.yml]: :open_mouth:

Comment (RamyAlshurafa) [reply .travis.yml]: ?

Comment (RamyAlshurafa) [reply .travis.yml]: @ashatat
can you check this PR
It cannot read the process. Env

File: client/src/components/Common/Navbar/index.js [hunk 2]
```diff
@@ -70,7 +70,7 @@ class Navbar extends Component {
   // RENDERING IS BASED ON KNOWING IF LOGGEDIN AND THE TYPE OF USER
 
   menuButtonClick = async e => {
-    const signOutResult = await axios.get("api/sign-out");
+    const signOutResult = await axios.get("/api/sign-out");
```
Comment (ashatat) [inline client/src/components/Common/Navbar/index.js]: should import it from apiRoutes

File: server/controllers/dashboard/hostDashboard.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    if (bookings[0]) {
+      const [{ intern: { profile: { profileImage } } }] = bookings;
+      generateFileURL(profileImage);
+    }
+
+    if (profile && profile.profileImage) generateFileURL(profile.profileImage);
+
+    return res.json({ ...dashboardData });
+  } catch (err) {
+    return next(err);
```
Comment (ashatat) [inline server/controllers/dashboard/hostDashboard.js]: use boom

File: server/controllers/payments/withdrawRequest.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+const withdrawRequest = async (req, res, next) => {
+  const {
+    amount, bankName, bankSortCode, bankNumber,
+  } = req.body;
+  const { role, account, _id } = req.user;
+
+  // check for user role
+  if (role !== "host" && role !== "superhost") {
+    return next(boom.unauthorized());
```
Comment (ashatat) [inline server/controllers/payments/withdrawRequest.js]:
```suggestion
    return next(boom.forbidden());
```

File: server/database/dbConnection.js [hunk 5]
```diff
@@ -15,10 +16,13 @@ if (process.env.NODE_ENV === "test") {
...
 }
 
-// create DB connection
-const dbConnection = () => mongoose.connect(mongoURI, {
+
+/**
+ * create DB connection
+ * @param {Boolean} useAtlas - true to use atlas DB
+ */
+const dbConnection = useAtlas => mongoose.connect(useAtlas ? mongoURIAtlas : mongoURI, {
```
Comment (ashatat) [inline server/database/dbConnection.js]: cool idea :)

File: client/src/components/Pages/HostDashboard/Content.js [hunk 6]
```diff
@@ -0,0 +1,461 @@
...
+              </BookingsTableWrapper>
+            ) : (
+              <Empty description="You didn't make any withdraw request" />
+            )}
+          </SectionWrapperContent>
+        </Col>
+      </ContentWrapper>
+      <div>
+        <Modal
+          // title="Basic Modal"
```
Comment (RamyAlshurafa) [inline client/src/components/Pages/HostDashboard/Content.js]: to me :)
remove the commented code

**Pull request merged**